src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2721,18 +2721,15 @@ public IEnumerable<CompletionResult> CompleteArgument(
         /// <param name="filePath">The file path to get verbs.</param>
         /// <param name="wordToComplete">The word to complete.</param>
         /// <returns>List of file verbs to complete.</returns>
-        private static IEnumerable<string> CompleteFileVerbs(string filePath, string wordToComplete)
+        private static IEnumerable<CompletionResult> CompleteFileVerbs(string filePath, string wordToComplete)
         {
-            var verbPattern = WildcardPattern.Get(wordToComplete + "*", WildcardOptions.IgnoreCase);
-
             string[] verbs = new ProcessStartInfo(filePath).Verbs;
 
-            foreach (string verb in verbs)
+            foreach (CompletionResult result in CompletionCompleters.EnumerateQuotedAndUnquotedCompletionText(
+                wordToComplete,
+                possibleCompletionValues: verbs))
             {
-                if (verbPattern.IsMatch(verb))
-                {
-                    yield return verb;
-                }
+                yield return result;
```

Comment:
We can return `CompletionCompleters.EnumerateQuotedAndUnquotedCompletionText()` because there aren't other yield-s.

Reply:
Thanks @iSazonov. I have also fixed this with some other instances.